.github/workflows/static_checks_and_tests.yml

```
@@ -0,0 +1,53 @@
+name: static checks and tests
+on:
+  pull_request:
+    branches: [ main ]
+  push:
+    branches: [ main ]
+jobs:
+  shfmt-check:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+    - name: Setup shfmt
+      run: |
+        wget 'https://github.com/mvdan/sh/releases/download/v3.4.3/shfmt_v3.4.3_linux_amd64' -O shfmt
```

```suggestion
        wget 'https://github.com/mvdan/sh/releases/download/v3.9.0/shfmt_v3.9.0_linux_amd64' -O shfmt
```
update to latest [v3.9.0](https://github.com/mvdan/sh/releases/tag/v3.9.0) release

---

would also be good to delete the rogue binary https://github.com/redpanda-data/chaos/blob/main/shfmt so it is not confusing to users of this repo